### DISCUSSION TITLE
jaeger-collection in cpu load only one pod, second pod is idle

### DISCUSSION THREAD

**Discussion by osipovdaniil** [jaegertracing/jaeger#3409] [Q&A]
Hi! I have jaeger version **1.20.0**, `jaeger-agent` as daemonset, `jaeger-collector` has 2 replicas
I see that the `jaeger-collector` under load only one pod, second pod is idle.
I want see uniform load on jaeger-collector pods.

---

First pod jaeger-collector
![Снимок экрана 2021-11-25 в 15 16 51](https://user-images.githubusercontent.com/73994399/143441563-320a3d27-1b27-4334-9c4e-1760cc86aa77.png)

Second pod jaeger-collector
![Снимок экрана 2021-11-25 в 15 17 02](https://user-images.githubusercontent.com/73994399/143441598-287a27c3-cf09-4eb8-9fd5-228553171c7c.png)

Any ideas? Why is that?

**Comment by jpkrohling** [comment]
How are you configuring the agent? You should be using the round-robin balancer, with the DNS protocol, pointing to a headless service backed by collector pods. This way, the agent will properly balance the requests to all IPs discovered during an initial query to the A record of the DNS entry.

Note that scaling up collectors won't have an effect until the agent is restarted. See #1678 for more details.

**Reply by osipovdaniil** [reply]
I use service clusterIP for collector.
I will try use headless-service for that. Thank's!
